test/dynamo/test_dynamic_shapes.py

```diff
@@ -26,7 +27,7 @@
 
 def make_dynamic_cls(cls):
     return make_test_cls_with_patches(
-        cls, "DynamicShapes", "_dynamic_shapes", ("dynamic_shapes", True)
+        cls, "DynamicShapes", "_dynamic_shapes", (config, "dynamic_shapes", True)
```

Comment:
`make_test_cls_with_patches` accepts the module name now, so I could pass more patches in the inductor test file.